Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Suspicious_Hidden_User_Created_Test.yml

```diff
@@ -1,1116 +1,3103 @@
+      version: -1
+      name: 'Search for login events '
+      description: |-
+        Execute an XQL query and retrieve results of an executed XQL query API. The command will be executed every 10 seconds until results are retrieved or until a timeout error is raised.
+        When more than 1000 results are retrieved, the command will return a compressed gzipped JSON format file,
+        unless the argument 'parse_result_file_to_context' is set to true and then the results will be extracted to the context.
+      script: '|||xdr-xql-generic-query'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#error#':
+      - "9"
+      '#none#':
+      - "60"
+    scriptarguments:
+      query:
+        simple: |-
+          preset = xdr_login_events
+          | alter dst_identity = login_data_dst_normalized_user -> username,
+            identity_type = login_data_dst_normalized_user -> identity_type
+          | filter outcome = "SUCCESS"
+          | filter lowercase(dst_identity) = lowercase("${Core.OriginalAlert.raw_abioc.event.action_evtlog_data_fields.TargetUserName}")
+          | fields dst_identity, identity_type
+      query_name:
+        simple: Authentication logs for the hidden username
+      time_frame:
+        simple: between ${QueryStartTime} and ${QueryEndTime}
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: errorPath
+    view: |-
+      {
+        "position": {
+          "x": -558,
+          "y": 1174
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "50":
+    id: "50"
+    taskid: 92ed7869-d2e9-44ba-b96d-255c3cb08266
+    type: regular
+    task:
+      id: 92ed7869-d2e9-44ba-b96d-255c3cb08266
+      version: -1
+      name: Search for share access/service install
+      description: |-
+        Execute an XQL query and retrieve results of an executed XQL query API. The command will be executed every 10 seconds until results are retrieved or until a timeout error is raised.
+        When more than 1000 results are retrieved, the command will return a compressed gzipped JSON format file,
+        unless the argument 'parse_result_file_to_context' is set to true and then the results will be extracted to the context.
+      script: '|||xdr-xql-generic-query'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#error#':
+      - "9"
+      '#none#':
+      - "61"
+    scriptarguments:
+      query:
+        simple: |-
+          dataset = xdr_data
+          | filter event_type = ENUM.EVENT_LOG
+            and action_evtlog_event_id in (5140, 7045)
+          | alter Host_Name = lowercase(actor_process_image_name)
+          | alter Source_Host = coalesce(agent_hostname, actor_process_image_name, action_process_image_name)
+          | filter Source_Host = "${alert.hostname}"
```

Comment:
lowercase(Source_Host) = lowercase("${alert.hostname})
something like this is better so we ensure we compare lowercase to lowercase (if we use lowercase just on one, we might have a problem with the other)